lib/iris/fileformats/_nc_load_rules/helpers.py

```diff
@@ -440,10 +440,13 @@ def build_mercator_coordinate_system(engine, cf_grid_var):
     longitude_of_projection_origin = getattr(
         cf_grid_var, CF_ATTR_GRID_LON_OF_PROJ_ORIGIN, None
     )
+    standard_parallel = getattr(
+        cf_grid_var, CF_ATTR_GRID_STANDARD_PARALLEL, None
+    )
```

Comment:
I take it this change is because you spotted that the `Mercator` class now has a `standard_parallel` parameter, rather than it being linked to `false_easting`/`false_northing`?

(I'm reviewing this PR with minimal knowledge of coordinate systems!)

Reply:
Yep